extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaNativeContainer.java

```diff
@@ -59,10 +57,13 @@ protected void containerIsStarting(InspectContainerResponse containerInfo, boole
             cmd += " " + additionalArgs;
         }
 
-        //noinspection OctalInteger
-        copyFileToContainer(
-                Transferable.of(cmd.getBytes(StandardCharsets.UTF_8), 0777),
-                STARTER_SCRIPT);
+        // docker exec since docker cp doesn't work with kubedock yet
+        try {
+            execInContainer("sh", "-c",
+                    String.format("echo -e \"%s\" >> %s && chmod 777 %s", cmd, STARTER_SCRIPT, STARTER_SCRIPT));
```

Comment:
The following might be a little less confusing (untested):
```suggestion
                    String.format("echo -e \"%1$s\" >> %2$s && chmod 777 %2$s", cmd, STARTER_SCRIPT));
```

Reply:
Thanks